lib/UserManagement.php

```diff
@@ -22,20 +22,24 @@ class UserManagement
      * @param string|null $firstName The first name of the user.
      * @param string|null $lastName The last name of the user.
      * @param boolean|null $emailVerified A boolean declaring if the user's email has been verified.
+     * @param string|null $passwordHash The hashed password to set for the user.
+     * @param string|null $passwordHashType The algorithm originally used to hash the password. Valid values are bcrypt.
```

Comment:
```suggestion
     * @param string|null $passwordHashType The algorithm originally used to hash the password. Valid values are `bcrypt`, `ssha`, and `firebase-scrypt`.
```

The `updateUser` docs that this was copied from needs an update as well.